src/pages/comparison/comparison.tsx

```diff
@@ -200,16 +260,30 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
         onRequestTransitionToTileset={() => {
           setNeedTransitionToTileset(true);
         }}
+        disableButtonHandler={disableButtonHandlerLeft}
+        onTilesetLoaded={() => loadManagerRef.current.resolveLeftSide()}
+        onStopTimer={() => loadManagerRef.current.stoptTimer()}
       />
-
       <Devider layout={layout} />
+      <CompareButton
+        compareButtonMode={compareButtonMode}
+        downloadStats={
+          compareButtonMode === CompareButtonMode.Start &&
+          loadManagerRef.current.haveBeenCompared
```

Comment:
You don't need this flag in the manager. "loaded" event is sufficient to know that there are some results